eth/api_backend.go

```diff
@@ -49,6 +49,9 @@ type EthAPIBackend struct {
 	//
 	// hex node id from node public key
 	hexNodeId string
+
+	// if set diables the time out for eth_call
+	timeOutForCall int
```

Comment:
i would use the type: `time.Duration` here and transform arg value in the `setQuorumConfig()`

Reply:
Yes agree. Will save multiple transformations. Changed.